tests/functional/spec/components/radio/radio_numeric_detail_answers.spec.js

```diff
@@ -0,0 +1,116 @@
+const helpers = require('../../../helpers');
+
+const RadioNumericDetailPage = require('../../../generated_pages/radio_numeric_detail_answers/radio-numeric-detail.page');
+const SummaryPage = require('../../../generated_pages/radio_numeric_detail_answers/summary.page');
+
+describe('Radio with a numeric "detail_answer" option', function() {
+
+  const radio_schema = 'test_radio_numeric_detail_answers.json';
+
+  it('Given a numeric detail answer options are available, When the user clicks an option, Then the detail answer input should be visible.', function() {
+    // Given
+    return helpers.openQuestionnaire(radio_schema).then(() => {
```

Comment:
Same as above, opening the questionnaire should be done in a beforeEach on the describe